modules/packages/ZMQ.chpl

```diff
@@ -493,7 +493,10 @@ module ZMQ {
       this.home = here;
       this.complete();
       if this.ctx == nil {
-        var errmsg = createStringWithNewBuffer(zmq_strerror(errno));
+        var errmsg: string;
+        try! {
+          errmsg = createStringWithNewBuffer(zmq_strerror(errno));
```

Comment:
In this kind of setting, it would make more sense to escape non-UTF-8 characters

Reply:
Right.

I also taught about using `bytes` here. For such short-lived strings that the library-writer knows to be utf8 (probably it is even ASCII in this case), using `bytes` can be a reasonable approach.